screens/Settings/SettingScreen.tsx

```diff
@@ -86,11 +88,7 @@ export const SettingScreen: React.FC<
                   </Text>
                 </Column>
               </Pressable>
-              <ReceivedCards
-                isVisible={false}
-                service={undefined}
-                vcItemActor={undefined}
-              />
+              <ReceivedCards />
```

Comment:
@srikanth716 after sharing the vc if we immediately click on Receive card of settings screen it is redirecting the user to history screen instead of showing the QR code. Please look into this

Reply:
Since @Anil-kumar-Majji has developed this receive and received cards, I have connected with him. He is saying that is an known issue and he is working on it. This changes are only for this ticket.  

@swathi Shall I fix that and include in this pr ?

Reply:
@PuBHARGAVI redirecting users back to QR code is the issue which has to be fixed but not part of this story